iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/ResourceVisitor.java

```diff
@@ -274,7 +293,12 @@ public PresentListProperty(CheckContext ctx, Block block, String name, Attribute
       
       @Override
       public void reportItemsWhichMatch(Predicate<ExpressionTree> predicate, String message, SecondaryLocation... secondaries) {
-        items.stream().filter(predicate).forEach(item -> ctx.reportIssue(item, message, Arrays.asList(secondaries)));
+        streamItemsWhichMatch(predicate).forEach(item -> ctx.reportIssue(item, message, Arrays.asList(secondaries)));
+      }
+
+      @Override
+      public Stream<ExpressionTree> streamItemsWhichMatch(Predicate<ExpressionTree> predicate) {
```

Comment:
[subjective - not an issue, just sharing thoughts]:
I start to realise our taste for naming methods is different.
I'd name this `streamItemsWhich(..)`, `itemsWhich(..)` or `itemsFiltered(..)`
Or skip introducing the method at alltogether just go for `items.stream().filter(predicate)` which is both (1)equivalent and (2) easy enough on the dev's eyes :)